external_deps/build.sh

```diff
@@ -216,19 +264,18 @@ build_zlib() {
 
 	"${download_only}" && return
 
-	cd "${dir_name}"
+	local zlib_cmake_args=(-DCMAKE_C_FLAGS="${CFLAGS} -DZLIB_CONST")
+
 	case "${PLATFORM}" in
-	windows-*-*)
-		LOC="${CFLAGS}" make -f win32/Makefile.gcc PREFIX="${HOST}-"
-		make -f win32/Makefile.gcc install BINARY_PATH="${PREFIX}/bin" LIBRARY_PATH="${PREFIX}/lib" INCLUDE_PATH="${PREFIX}/include" SHARED_MODE=1
-		;;
-	*)
-		# The default -O3 is dropped when there's user-provided CFLAGS.
-		CFLAGS="${CFLAGS} -O3" ./configure --prefix="${PREFIX}" --libdir="${PREFIX}/lib" --static --const
-		make
-		make install
+		windows-*-*)
```

Comment:
zlib now produces both static and shared libs for mingw platforms. Previously it only produced DLLs.